openfisca_aotearoa/tests/social_security/covid_19_wage_subsidy.yaml

```diff
@@ -0,0 +1,26 @@
+- name: Sole trader, eligible for wage subsidy
```

Comment:
These tests are both failing in CI - with the errors:

```
AssertionError: In test 'Sole trader, does not have IRD number, not eligible for wage subsidy', in file '/root/project/openfisca_aotearoa/tests/social_security/covid_19_wage_subsidy.yaml', covid_19_wage_subsidy__eligibility@2020-6: [0.] differs from 1.0 with an absolute margin [1.] > 0
```

Reply:
As below - some debugging with @JacOng17 reveals that it's because of the missing `located_in_NZ` parameter. If you make those changes, the tests should pass @asgharkazi @sarafalamaki